src/components/icons/create-styled-icon.js

```diff
@@ -1,7 +1,7 @@
 import PropTypes from 'prop-types';
 import invariant from 'tiny-invariant';
 import styled from '@emotion/styled';
-import vars from '../../../materials/custom-properties';
+import vars from '@commercetools-uikit/custom-properties';
```

Comment:
Should we name this package something like `@commercetools-uikit/design-system` or `@commercetools-uikit/design-tokens`? 🤔